DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/movement-sensor/accel-adxl345-micro-rdk.md

If you are using a [Viam Rover](/appendix/try-viam/), this is the accelerometer on it.

> Make sure to physically connect your movement sensor to your machine's microcontroller and turn it on.

Reviewer: ```suggestion
Physically connect your movement sensor to your microcontroller and power both on.
```
